tests/by-util/test_ls.rs

```diff
@@ -814,8 +814,8 @@ fn test_ls_zero() {
     let scene = TestScenario::new(util_name!());
     let at = &scene.fixtures;
     at.mkdir("0-test-zero");
-    at.touch(&at.plus_as_string("2-test-zero"));
-    at.touch(&at.plus_as_string("3-test-zero"));
+    at.touch(at.plus_as_string("2-test-zero"));
```

Comment:
These changes seems to be unrelated. Can you move them into a different commit? or PR

Reply:
They are fixing clippy warnings because I changed the signature of `touch` to take `AsRef<Path>` instead of `&str`.

Reply:
oh, right, thanks

Reply:
could you please fix the other below ?
tests/by-util/test_chown.rs
test_install etc